recipes/astroid/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
 
   run:
     - python
+    - setuptools
```

Comment:
I don't see any imports outside of `setup.py`. We may need to fix the entrypoints though.

ref: https://github.com/PyCQA/pylint/search?utf8=%E2%9C%93&q=setuptools